packages/scm/src/browser/scm-widget.tsx

```diff
@@ -152,6 +152,10 @@ export class ScmWidget extends BaseWidget implements StatefulWidget {
 
     protected onActivateRequest(msg: Message): void {
         super.onActivateRequest(msg);
+        MessageLoop.sendMessage(this.commitWidget, msg);
```

Comment:
Why do we need to focus all of them? Only one thing can be focused in DOM. It looks correct what was before.

Reply:
Yes, I had not quite realized the intent behind onActivateRequest.  These changes are no longer needed now the code was moved to onAfterAttach and I have reverted the changes in scm-widget.